packages/ai-bot/tests/prompt-construction-test.ts

```diff
@@ -2778,6 +2778,107 @@ Attached files:
         '[Omitting previously suggested and rejected code change]\n',
     );
   });
+
+  test('Correctly handles server-side aggregations', async () => {
+    // sending older codeblocks back to the model just confuses it and wastes tokens
```

Comment:
This comment seems out of place. Copy pasta?

Reply:
ah yes, good spot. Had copied an example with mocked matrix files. I'll remove and merge.